### PR TITLE
Fix #2537: Add go back to site button

### LINKED ISSUE #2537: 'go back to website' button on authentication page
**Is your feature request related to a problem? Please describe.**
I would like to have 'Go back to page' on my admin page

**Describe the solution you'd like**
Add 'Go back to page' button to `/admin` authentication page

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by pniedzwiedzinski**
If this is a bug fix, make sure your description includes "fixes #xxxx", or
"closes #xxxx", where #xxxx is the issue number.

Please provide enough information so that others can review your pull request.
The first two fields are mandatory:
-->

**Summary**

Closes #2537. I added link to the '/' page. It might be good to take this link from `site_url` in `config.yml`, but as far as I know this entry is not required.

**Test plan**

image

**A picture of a cute animal (not mandatory but encouraged)**
image

### CODE REVIEW

**Comment by erquhart** [comment]
Thanks for opening! I'd be up for this if:

1. We show it optionally, based on the presence of `site_url`
2. Use `site_url` as the url
3. Just make it a gray text link: "<- Back to site", replacing the arrow with one of the svg icons
4. Create the component once and reuse
5. Document the extended behavior of `site_url`

**Comment by pniedzwiedzinski** [comment]
You mean something like this:
image

File: packages/netlify-cms-backend-bitbucket/src/AuthenticationPage.js [hunk 1]
```diff
@@ -80,6 +80,9 @@ export default class BitbucketAuthenticationPage extends React.Component {
           <React.Fragment>
             <LoginButtonIcon type="bitbucket" />
             {inProgress ? 'Logging in...' : 'Login with Bitbucket'}
+            {config.get('site_url') ? (
```
Comment (erquhart) [inline packages/netlify-cms-backend-bitbucket/src/AuthenticationPage.js]: These can be shrunk down: `{config.get('site_url') && <GoBackButton href={config.get('site_url')}/>}`

**Comment by pniedzwiedzinski** [comment]
image

**Comment by erquhart** [comment]
Looks like tests aren't passing.

**Comment by pniedzwiedzinski** [comment]
Should be good now

**Comment by erquhart** [comment]
Looks great, just noticed the GitHub backend isn't addressed, mind adding that qjuick?

**Comment by pniedzwiedzinski** [comment]
I have no idea why these checks are failing...

**Comment by erquhart** [comment]
Try rebasing onto the latest master.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by pniedzwiedzinski** [comment]
```bash
$ git fetch origin
$ git rebase origin/master
Current branch feature/2537 is up to date.
```
It's already on the newest

**Comment by pniedzwiedzinski** [comment]
Now e2e tests are failing :/ 
https://github.com/pniedzwiedzinski/netlify-cms/commit/1fbd2bb3315b6aede1dbfa68ea81eee39cc75de2/checks

**Comment by pniedzwiedzinski** [comment]
What's next?

**Comment by erquhart** [comment]
Looks good, thanks @pniedzwiedzinski 👍

**Pull request merged**